frontend/src/component/feature/FeatureStrategy/FeatureStrategyEmpty/FeatureStrategyEmpty.tsx

```diff
@@ -118,77 +138,92 @@ export const FeatureStrategyEmpty = ({
         otherAvailableEnvironments && otherAvailableEnvironments.length > 0;
 
     return (
-        <div className={styles.container}>
-            <div className={styles.title}>
-                You have not defined any strategies yet.
-            </div>
-            <p className={styles.description}>
-                Strategies added in this environment will only be executed if
-                the SDK is using an{' '}
-                <Link to="/admin/api">API key configured</Link> for this
-                environment.
-            </p>
-            <Box
-                sx={{
-                    w: '100%',
-                    display: 'flex',
-                    flexWrap: 'wrap',
-                    gap: 2,
-                    alignItems: 'center',
-                    justifyContent: 'center',
-                }}
-            >
-                <FeatureStrategyMenu
-                    label="Add your first strategy"
-                    projectId={projectId}
-                    featureId={featureId}
-                    environmentId={environmentId}
-                    matchWidth={canCopyFromOtherEnvironment}
-                />
-                <ConditionallyRender
-                    condition={canCopyFromOtherEnvironment}
-                    show={
-                        <CopyButton
-                            environmentId={environmentId}
-                            environments={otherAvailableEnvironments.map(
-                                environment => environment.name
-                            )}
-                            onClick={onCopyStrategies}
-                        />
-                    }
-                />
-            </Box>
-            <Box sx={{ width: '100%', mt: 3 }}>
-                <SectionSeparator>Or use a strategy template</SectionSeparator>
-            </Box>
-            <Box
-                sx={{
-                    display: 'grid',
-                    width: '100%',
-                    gap: 2,
-                    gridTemplateColumns: { xs: '1fr', sm: '1fr 1fr' },
-                }}
-            >
-                <PresetCard
-                    title="Standard strategy"
-                    Icon={getFeatureStrategyIcon('default')}
-                    onClick={onAddSimpleStrategy}
-                    projectId={projectId}
-                    environmentId={environmentId}
+        <>
+            <ChangeRequestDialogue
```

Comment:
You pass some props that are then passed again internally. It's more idiomatic to use component composition in React to avoid passing data inside. You do it by passing components as children instead of passing data and building child components inside your parent component. It's called slots not props. More info: https://www.youtube.com/watch?v=3XaXKiXtNjw

Reply:
How's this?